utils/dump_leaderboard.go

```diff
@@ -114,25 +128,43 @@ func main() {
 		members = append(members, member)
 	}
 
+	if *sortFlag == "default" {
+		sort.Sort(ByName(members))
+	} else if *sortFlag == "stars" {
+		sort.Sort(ByStars(members))
+	} else if *sortFlag == "names" {
+		sort.Sort(ByName(members))
+	} else {
+		log.Fatal(fmt.Sprintf("invalid sort: %s. Allowed: %s", *sortFlag, allowedSorts))
+	}
+
 	// Build the completions for the given day.
-	results := map[int][]Result{
-		1: []Result{},
-		2: []Result{},
+	var results [25]map[int][]Result
```

Comment:
I think you want 26, here and above. On 154 (and presumably elsewhere) you're indexing with the day number, which is 1-25. It works now because we're not at Christmas yet, but on Christmas morning it should explode.

Reply:
Updated. We saved Christmas and Advent of Code isn't even over yet!